android/app/src/main/java/com/mattermost/MainApplication.java

```diff
@@ -82,4 +83,9 @@ public IPushNotification getPushNotification(Context context, Bundle bundle, App
             new JsIOHelper()
     );
   }
+
+  @Override
+  protected List<ReactPackage> getPackages() {
+    return Arrays.<ReactPackage>asList(new KeyboardInputPackage(MainApplication.this));
```

Comment:
I just followed the installation instructions from `react-native-keyboard-input` but I'll check the approach you mentioned